src/profile-logic/call-tree.js

```diff
@@ -100,8 +100,7 @@ export class CallTree {
    */
   preloadChildrenCache() {
     if (!this._isChildrenCachePreloaded) {
-      this._children.clear();
```

Comment:
In the previous code, we were "clearing" the map here, do you know why ? We're not clearing the array anymore now...

Reply:
Yes, it was me who wrote that `clear` call. It was just there to get a clean slate to work on, not realizing that it wasn't needed since everything would get wiped out anyway. So now I just removed it when I read this code again.